DigitalLearningSolutions.Data/Services/PasswordResetService.cs

```diff
@@ -82,7 +83,7 @@ public async Task InvalidateResetPasswordForEmailAsync(string email)
             }
         }
 
-        private string GenerateResetPasswordHash(User user)
+        public string GenerateResetPasswordHash(User user)
         {
             string hash = Guid.NewGuid().ToString();
 

```

Comment:
I think the email generation file structure should be consistent between Reset & Set password so worth shuffling things around